README.md

```diff
@@ -44,7 +47,7 @@ Android Engineer Interview Questions
  - What does setRetainInstance do and how you can avoid it?
  - What callbacks trigger when a Dialog opens up? Conside both case, dialog attached from same activity/fragment and other activity/fragment.
  - What does `launchWhenCreated`, `launchWhenStarted`, and `launchWhenResumed` functions do?
- - 
+ - What are ‘activities’? Describe the lifecycle of an activity.
```

Comment:
Nice.
You can add a question about the fragment's lifecycle as well. 
eg How does the activity's lifecycle affects the hosted fragment? Can the fragment instance be retained over config change etc